build.gradle

```
@@ -45,10 +45,10 @@ ext {
     buildToolsVersion = "29.0.2"
     compileSdkVersion = 29
     minSdkVersion = 16
-    nav_version = "2.3.0"
     targetSdkVersion = 29
 
     awsSdkVersion = '2.16.12'
+    navigationversion = "2.3.0"
```

```suggestion
    navigationversion = '2.3.0'
```
Single quotes, not double.